src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -119,12 +121,19 @@ public int CreateIndex(IndexDefinition definition)
                 definition.RemoveDefaultValues();
 
                 ValidateAnalyzers(definition);
+                var createReplacementIndex = false;
+                var replacementIndexName = Constants.Documents.Indexing.SideBySideIndexNamePrefix + definition.Name;
+                Index replacementIndex;
 
                 switch (GetIndexCreationOptions(definition, existingIndex))
                 {
                     case IndexCreationOptions.Noop:
+                        if (_indexes.TryGetByName(replacementIndexName, out replacementIndex))
```

Comment:
I am deleting the replacement index because the new definition is like the original one